src/header/interfaces.ts

```diff
@@ -41,6 +41,11 @@ export interface HeaderProps extends BaseComponentProps {
    * Area next to the heading to display an Info link.
    */
   info?: React.ReactNode;
+  /**
+   * Determines if the heading (specified with the `children` property) should be truncated with ellipsis when there is not enough space.
+   * Defaults to `false`.
+   */
+  truncateHeading?: boolean;
```

Comment:
Naming. 

1. We have similar props on button component (called `wrapText`) and table (`wrapLines`). So the new property should follow this pattern
2. The content of default slot is called `title` (see display name in the [API docs](https://github.com/cloudscape-design/components/blob/1b69f03545f9f0aaf88fb685e28efabf1ca05140/src/header/interfaces.ts#L10) and also [usage guidelines](https://cloudscape.design/components/header/?tabId=usage)).

Based on these two, I suggest `wrapTitle` for the name

Reply:
P.S. you may also look at API docs for respective components and reuse some of the wording, but I do not have much opinion there

Reply:
Thanks. 

The reason I used `heading` and not `title` is because title also contains counter and info link that are not truncated.

The reason I used `truncate` and not `wrap` is for default value for that property to be `false`.

I am happy to challenge both decisions - I'll bring that to the team review.